pom.xml

```diff
@@ -211,16 +225,28 @@
             <version>RELEASE82</version>
         </dependency>
         <dependency>
-            <groupId>com.github.eirslett</groupId>
-            <artifactId>frontend-maven-plugin</artifactId>
-            <version>1.6</version>
+            <groupId>org.netbeans.external</groupId>
+            <artifactId>net-java-html-boot</artifactId>
+            <version>RELEASE82</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.netbeans.api</groupId>
+            <artifactId>org-netbeans-api-htmlui</artifactId>
+            <version>RELEASE82</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.external</groupId>
+            <artifactId>net-java-html</artifactId>
+            <version>RELEASE82</version>
+        </dependency>
     </dependencies>
     <properties>
         <project.author>Chris2011</project.author>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <netbeans.compile.on.save>none</netbeans.compile.on.save>
```

Comment:
This is now needed because of usage of `html4j-maven-plugin`.

Reply:
ok thx.